### PR TITLE
Mark tests as pending. Fixes issue 660

### PR DESCRIPTION
**Pull request by theCrab**
Mark the auto generated tests as skipped/pending. Fixes issue #660

### CODE REVIEW

**Comment by theCrab** [comment]
@jodosha did I do something wrong? Is master branch the correct place?
```rb
RSpec.describe Hanami::VERSION do
  it 'returns current version' do
    expect(subject).to eq('1.0.0.alpha1')
  end
end
```

**Comment by cllns** [comment]
<3 thank you! :)

File: lib/hanami/generators/action/view_spec.rspec.tt [hunk 1]
```diff
@@ -7,6 +7,8 @@ RSpec.describe <%= config[:app] %>::Views::<%= config[:controller] %>::<%= confi
   let(:rendered)  { view.render }
 
   it 'exposes #foo' do
+    pending 'This is an auto-generated test. You should edit or delete it.'
+
     expect(view.foo).to eq exposures.fetch(:foo)
```
Comment (AlfonsoUceda) [inline lib/hanami/generators/action/view_spec.rspec.tt]: @theCrab I think you should add the `# Example` here or delete above, we should have the same code.

**Comment by AlfonsoUceda** [comment]
thanks @theCrab this looks perfect, I think this can be merge, what do you think @jodosha

**Comment by AlfonsoUceda** [comment]
@theCrab could you fix the problems? thanks ;)

**Comment by TiteiKo** [comment]
👍  thanks!

**Comment by AlfonsoUceda** [comment]
@theCrab could you resolve the conflict? after that I'll merge it.

**Comment by theCrab** [comment]
@AlfonsoUceda Looks good?

**Comment by AlfonsoUceda** [comment]
@theCrab perfect!

**Pull request merged**